misc/userscripts/open_download

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash -e
+# Both standalone script and qutebrowser userscript that opens a rofi menu with
+# all files from the download director and opens the selected file. It works
+# both as a userscript and a standalone script that is called from outside of
+# qutebrowser.
+#
+# Suggested keybinding (for "show downloads"):
+# spawn --userscript ~/.config/qutebrowser/open_download
+#     sd
+#
+# Requirements:
+#  - rofi (in a recent version)
+#  - xdg-open and xdg-mime
+#  - You should configure qutebrowser to download files to a single directory
+#  - It comes in handy if you enable remove-finished-downloads. If you want to
+#    see the recent downloads, just press "sd".
+#
+# Thorsten Wißmann, 2015 (thorsten` on freenode)
+# Any feedback is welcome!
+
+# open a file from the download directory using rofi
+DOWNLOAD_DIR=${DOWNLOAD_DIR:-$HOME/downloads}
```

Comment:
Would you mind changing this to `~/Downloads`? That's kind of an usual place for downloads IMHO.